test-Pfaffian.jl

```diff
@@ -90,34 +91,34 @@ println()
 println("Testing complex methods:")
 println()
 
-x = rand(Complex128, N)
+x = rand(ComplexF64, N)
 
 v, tau, alpha = Householder(x)
-v_w, tau_w, alpha_w = pfaffian[:householder_complex](x)
+v_w, tau_w, alpha_w = pf.householder_complex(x)
 
 @show maximum(abs, v - v_w)
 @show abs(tau - tau_w)
 @show abs(alpha - alpha_w)
 println()
 
-A = rand(Complex128, (N, N))
-A = A - A.'
+A = rand(ComplexF64, (N, N))
+A = A - A'
 
 T, Q = skew_tridiagonalize(A)
-T_w, Q_w = pfaffian[:skew_tridiagonalize](A)
+T_w, Q_w = pf.skew_tridiagonalize(A)
```

Comment:
This currently throws
```AssertionError()
  File "/Users/mishmash/opt/anaconda3/envs/pfaffiantests37/lib/python3.7/site-packages/pfapack/pfaffian.py", line 93, in skew_tridiagonalize
    assert abs((A + A.T).max()) < 1e-14
```
since `A'` is conjugate transpose. Should probably use `transpose` throughout to make the antisymmetric matrices.

Reply:
And also in the corresponding antisymmetric-checking asserts in the Julia code.

Reply:
@mishmash do you mean in the corresponding Julia code right? If so yes, you should use transpose(A) and not the ' as it does the conjugation too.

Reply:
Yes, in the Julia code we need to be careful about transpose vs. conjugate transpose throughout.